run

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Created this because I was lazy but we could provide a bash scripts  that does the same thing but gets an archive as param.

so we can run the tool like: `./tosca2vsphere sampleapp.zip` and it is not necessary to type `java -jar .... `